docs/autofish.rst

```diff
@@ -0,0 +1,42 @@
+autofish
+========
+
+.. dfhack_tool::
+    :summary: Keep a specific quantity of fish in your fortress.
+    :tags: fort auto fps jobs
+
+This script makes managing how much fish you keep around automatic. It tries to
+maintain a configured stock level of raw and/or prepared fish, partially to keep
+item quantities from ballooning out of control, and partly to try and prevent
+collecting too many rotten fish.
+
+Usage
+-----
+``enable autofish``
+    Enable the script
+``disable autofish``
+    Disable the script
+``autofish status``
+    Show the current status of the script, your configured values, and whether
+    or not fishing is currently enabled.
+``autofish <max> [min] [<options>]``
+    Change autofish settings.
+
+    ``max`` (default: 100) controls the maximum amount of fish you  want to keep
+    on hand in your fortress. Fishing will be disabled when the amount of fish
+    goes above this value.
+
+    ``min`` (default: 50) controls the minimum fish you want before restarting
+    fishing.
+
+    Use ``--include-raw``(``-r``) (default: off) to toggle letting the script
+    also count your raw fish as part of your quota. Use it a second time to
+    disable this.
+
+Examples
+--------
+
+``autofish -r 150``
```

Comment:
I think the first example should just be `enable autofish`, since that is the simplest use case, and is all a user really needs to do.